ivy/functional/frontends/torch/reduction_ops.py

```diff
@@ -180,6 +180,22 @@ def min(*input, dim=None, keepdim=False, out=None):
         )
 
 
+@numpy_to_torch_style_args
+@to_ivy_arrays_and_back
+def mode(*input, dim=None, keepdim=False, out=None):
+    if dim is None:
+        input = ivy.reshape(input, [-1])
+        dim = 0
+    unique_values, counts = ivy.unique(input, return_counts=True, axis=dim)
```

Comment:
ivy doesn't have this function, but we have `ivy.unique_counts`, `ivy.unique_inverse`, `ivy.unique_all` or `ivy.unique_values` which one of them should be the one you're looking for